tsconfig-all.json

```diff
@@ -0,0 +1,86 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "rootDir": "./",
+        "declaration": false
+    },
+    "include": [
+        "packages/**/*.ts",
+        "tools/**/*.ts",
+        "projects/story-decorator/**/*.ts",
+        "projects/vrt-compare/**/*.ts",
+        "test/lit-helpers.ts"
+    ],
+    "exclude": [
+        "packages/*/node_modules/**/*.ts",
+        "tools/*/node_modules/**/*.ts"
+    ],
+    "references": [
```

Comment:
Would it make sense to generate these automatically rather than manually curate them?

Reply:
This is a great idea. I've created https://github.com/adobe/spectrum-web-components/issues/2162 to track. With the relative stability of the library I don't think we should block on getting the perf benefits here.